.devcontainer/devcontainer.json

```diff
@@ -19,7 +19,7 @@
   },
   "updateContentCommand": "[ -f packages.txt ] && sudo apt update && sudo apt upgrade -y && sudo xargs apt install -y <packages.txt; [ -f requirements.txt ] && pip3 install --user -r requirements.txt; pip3 install --user streamlit; echo '✅ Packages installed and Requirements met'",
```

Comment:
wondering whether we should adjust this according to https://pixi.sh/latest/ide_integration/devcontainer/

Reply:
Since I am a happy PyCharm user, I don't really care about this. Happy to adjust it if @jaimergp agrees.

Reply:
You can have multiple devcontainer files, so feel free to add one backed by Pixi in a subsequent PR. It should look like:

```
.devcontainer/<name>/devcontainer.json
.devcontainer/<another-name>/devcontainer.json
```

Reply:
didn't know that, interesting 👀

Reply:
done in https://github.com/Quansight-Labs/conda-metadata-app/pull/12